src/store/store.ts

```diff
@@ -1,12 +1,20 @@
 import { configureStore } from '@reduxjs/toolkit';
 
-import exampleReducer from '@/store/exampleSlice';
+import exampleReducer from '@/store/slices/exampleSlice';
+import authReducer, { setAccessToken } from '@/store/slices/authSlice';
+
+const loadAccessToken = (): string => {
+  return localStorage.getItem('accessToken') || '';
```

Comment:
use redux-persist and this line will not be needed it have local storage underhood

Reply:
removed